main.py

```diff
@@ -45,159 +41,180 @@
+            debugging of data processing
+
+    """
     print("Starting to ETL...")
 
     client, credentials = get_sheets_client(
         credential_file_path=os.path.abspath(PATH_TO_SERVICE_ACCOUNT_KEY)
     )
 
-    cdc_beds_current_df = extract_cdc_beds_current_data()
-    cdc_beds_historical_df = extract_cdc_beds_historical_data(credentials=credentials)
+    # TODO(lbrown): Un-comment these when we find a path forward for CDC bed data.
+    # cdc_beds_current_df = extract_cdc_beds_current_data()
+    # cdc_beds_historical_df = extract_cdc_beds_historical_data(credentials=credentials)
+
+    # transformed_cdc_beds_df = transform_cdc_beds_data(
+    #     cdc_beds_current_df=cdc_beds_current_df,
+    #     cdc_beds_historical_df=cdc_beds_historical_df,
+    # )
+
+    # Upload category 3A data.
+    # criteria_3_summary_df = calculate_state_summary(
+    #     transformed_df=transformed_cdc_beds_df, columns=CRITERIA_3_SUMMARY_COLUMNS
+    # )
+    # post_dataframe_to_google_sheets(
+    #     df=criteria_3_summary_df,
+    #     workbook_key=CDC_CRITERIA_3_GOOGLE_WORKBOOK_KEY,
+    #     tab_name=STATE_SUMMARY_TAB_NAME,
+    #     credentials=credentials,
+    # )
+    #
+    # sleep_and_log()
+    #
+    # post_dataframe_to_google_sheets(
+    #     df=transformed_cdc_beds_df,
+    #     workbook_key=CDC_CRITERIA_3_GOOGLE_WORKBOOK_KEY,
+    #     tab_name="Historical Data",
+    #     credentials=credentials,
+    # )
+
     covidtracking_df = extract_covidtracking_historical_data()
     cdc_ili_df = extract_cdc_ili_data()
 
-    transformed_cdc_beds_df = transform_cdc_beds_data(
-        cdc_beds_current_df=cdc_beds_current_df,
-        cdc_beds_historical_df=cdc_beds_historical_df,
-    )
-
     transformed_cdc_ili_df = transform_cdc_ili_data(ili_df=cdc_ili_df)
 
     transformed_covidtracking_df = transform_covidtracking_data(
         covidtracking_df=covidtracking_df
     )
 
-    # Upload category 3A data.
-    criteria_3_summary_df = calculate_state_summary(
-        transformed_df=transformed_cdc_beds_df, columns=CRITERIA_3_SUMMARY_COLUMNS
-    )
-    post_dataframe_to_google_sheets(
-        df=criteria_3_summary_df,
-        workbook_key=CDC_CRITERIA_3_GOOGLE_WORKBOOK_KEY,
-        tab_name=STATE_SUMMARY_TAB_NAME,
-        credentials=credentials,
-    )
-
-    sleep_and_log()
-
-    post_dataframe_to_google_sheets(
-        df=transformed_cdc_beds_df,
-        workbook_key=CDC_CRITERIA_3_GOOGLE_WORKBOOK_KEY,
-        tab_name="Historical Data",
-        credentials=credentials,
-    )
-
-    sleep_and_log()
-
     # Upload summary for all states.
-    post_dataframe_to_google_sheets(
-        df=calculate_state_summary(
-            transformed_df=transformed_covidtracking_df, columns=STATE_SUMMARY_COLUMNS
-        ),
-        workbook_key=CDC_GUIDANCE_GOOGLE_WORKBOOK_KEY,
-        tab_name=STATE_SUMMARY_TAB_NAME,
-        credentials=credentials,
-    )
-
-    sleep_and_log()
+    if post_to_google_sheets:
```

Comment:
Nice addition – was just talking to @wesmwoo about potentially adding this flag with his changes but you beat us to it 😉 . Do you think it could be nice to write the CSVs locally if `post_to_google_sheets` is `False` – eg. using `pandas.DataFrame.to_csv`?

Reply:
Oooh, good idea. Maybe we leave that for another PR? I'm game for it though, would definitely help with debugging. we could refactor `post_dataframe_to_google_sheets` to accept a parameter that instead writes to a local file store.